packages/widgets/src/components/SearchBox/SearchBoxCrossIcon.tsx

```diff
@@ -0,0 +1,27 @@
+import { useTheme } from '@emotion/react';
+import { Icon } from '@ssa-ui-kit/core';
+
+export const SearchBoxCrossIcon = ({
+  onClick,
+}: {
+  onClick: React.MouseEventHandler<HTMLDivElement>;
+}) => {
+  const theme = useTheme();
+  return (
+    <div
+      role="button"
```

Comment:
It's better to use `<button />` instead of a div with the role "button" from the A11y standpoint. 
But it's up to you.